lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -629,13 +632,40 @@ private Optional<JourneyResponse> checkForProfileMatch(
         return Optional.empty();
     }
 
-    private JourneyResponse buildF2FNoMatchResponse(
+    private JourneyResponse buildAsyncCriNoMatchResponse(
+            AsyncCriStatus asyncCriStatus,
+            VerifiableCredentialBundle vcs,
+            Vot lowestGpg45ConfidenceRequested,
             boolean areGpg45VcsCorrelated,
             AuditEventUser auditEventUser,
             String deviceInformation,
             List<ContraIndicator> contraIndicators)
-            throws ConfigException, MitigationRouteException {
-        LOGGER.info(LogHelper.buildLogMessage("F2F return - failed to match a profile."));
+            throws ConfigException, MitigationRouteException, VerifiableCredentialException {
+        LOGGER.info(LogHelper.buildLogMessage("Async CRI return - failed to match a profile."));
+
+        if (DCMAW_ASYNC.equals(asyncCriStatus.cri())) {
+            if (asyncCriStatus.iat() == null) {
+                LOGGER.info(LogHelper.buildLogMessage("DCMAW async VC has no iat."));
+                return JOURNEY_ERROR;
+            }
+
+            var isExpired =
+                    DateUtils.toSecondsSinceEpoch(new Date()) > asyncCriStatus.iat() + (30 * 60);
+            if (isExpired) {
+                LOGGER.info(LogHelper.buildLogMessage("DCMAW async VC expired."));
+                return JOURNEY_ERROR;
```

Comment:
We probably want to kick off a new journey in this case (and for missing iat above?)